manifests/kustomize/README.md

```diff
@@ -4,35 +4,44 @@ This folder contains Kubeflow Pipelines Kustomize manifests for a light weight
 deployment. You can follow the instruction and deploy Kubeflow Pipelines in an
 existing cluster.
 
-## TL;DR
+To install Kubeflow Pipelines, you have several options.
+- Via [GCP AI Platform UI](http://console.cloud.google.com/ai-platform/pipelines).
+- Via an upcoming commandline tool.
+- Via Kubectl with Kustomize, it's detailed here.
+  - Community maintains a repo [here](https://github.com/e2fyi/kubeflow-aws/tree/master/pipelines) for AWS.
 
-Deploy latest version of Kubeflow Pipelines
+## Install via Kustomize
 
+Deploy latest version of Kubeflow Pipelines.
+
+It uses following default settings.
+- image: latest released images
+- namespace: kubeflow
+- application name: pipeline
+
+### Option-1 Install it to any K8s cluster
 ```
-export PIPELINE_VERSION=0.4.0
-kubectl apply -f https://storage.googleapis.com/ml-pipeline/pipeline-lite/$PIPELINE_VERSION/cluster-scoped-resources.yaml
-kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
-kubectl apply -f https://storage.googleapis.com/ml-pipeline/pipeline-lite/$PIPELINE_VERSION/namespaced-install.yaml
+kubectl apply -k cluster-scoped-resources/
+kubectl wait crd/applications.app.k8s.io --for condition=established --timeout=60s
+kubectl apply -k env/dev/
+kubectl wait applications/pipeline -n kubeflow --for condition=Ready --timeout=1800s
+kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80
 ```
+Now you can access it via localhost:8080
 
-Then get the Pipeline URL
-
+### Option-2 Install it to GCP with in-cluster PersistentVolumeClaim storage
 ```
+kubectl apply -k cluster-scoped-resources/
+kubectl wait crd/applications.app.k8s.io --for condition=established --timeout=60s
+kubectl apply -k env/gcp-dev/
+kubectl wait applications/pipeline -n kubeflow --for condition=Ready --timeout=1800s
 kubectl describe configmap inverse-proxy-config -n kubeflow | grep googleusercontent.com
 ```
 
-## Customization
-
-Customization can be done through Kustomize
-[Overlay](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#overlay).
-
-Note - The instruction below assume you installed kubectl v1.14.0 or later,
-which has native support of kustomize. To get latest kubectl, visit
-[here](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-
-### Deploy on GCP with CloudSQL and GCS
+### Option-3 Install it to GCP with CloudSQL & GCS-Minio managed storage
+It's suggested for production usage as data always is not stored to the cluster but CloudSQL & GCS.
 
-See [here](env/gcp/README.md) for more details.
+It's to be fixed soon in another PR.
```

Comment:
Probably highlight a little?
like `NOTE: this doesn't work now, will be fixed soon`?